README.md

```diff
@@ -27,6 +28,12 @@ Overview
 =========
 ![libjson-rpc-cpp logo](https://github.com/cinemast/libjson-rpc-cpp/blob/develop/dev/artwork/overview.png?raw=true)
 
+**TCP Connectors**
```

Comment:
This section seems a bit out of context here. Please use doxygen to express comments about modules. The README is already too blown up.